### PR TITLE
testsuite: handle job signal race in more tests

### PR DESCRIPTION
**Pull request by chu11**
Same fix as in #5302 and #5401 

also threw a increased timeout length for a test

### CODE REVIEW

**Comment by chu11** [comment]
re-pushed, with an extra timeout increase in a test

File: t/t2611-debug-emulate.t [hunk 1]
```diff
@@ -13,7 +13,7 @@ stop_tasks_test()     {
 
 test_under_flux 2
 
-TIMEOUT=10
+TIMEOUT=100
```
Comment (grondo) [inline t/t2611-debug-emulate.t:16]: The test at the end expects to fail with a timeout, so we don't want to blindly increase the timeout 10x in this test. (Did this actually succeed for you?)

The last test has:
```sh
        test_must_fail flux job wait-event -vt ${TIMEOUT} ${jobid} finish &&
```
In order to check that the job does _not_ finish after attach by `flux job attach --debug`. If you increase the timeout to 100 then the test will take a minimum of 100 seconds to run... and other timeouts may come into play ...

**Review by grondo — COMMENTED** [review]
One comment inline (probably the reason CI is failing...)

File: t/t2611-debug-emulate.t (hunk 1 above)
Comment (chu11) [reply t/t2611-debug-emulate.t:16]: hmmm, well it's failing for me now.  I have no idea how this passed for me before.  I clearly did something stupid before.  But yeah, lets update that one line to use a timeout of 2.

**Comment by chu11** [comment]
re-pushed with the suggested tweak from @grondo above, thanks!

**Review by grondo — APPROVED** [review]
Ok, LGTM!

**Pull request merged**